sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubOptions.cs

```diff
@@ -37,7 +38,29 @@ public EventHubOptions()
             InitialOffsetOptions = new InitialOffsetOptions();
```

Comment:
This is still required as opposed to exposing DefaultStartingPosition so that we can support constructing from config file.

Reply:
My vote would be to unify on this rather than providing differently named options for the same thing.

Reply:
Let's get rid of InitialOffsetOptions, you can read IConfiguration directly like this: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs#L41

Reply:
As discussed, we would need to use private reflection to get the internal properties of the EventPosition when options.Format() is called, and even then, round-tripping wouldn't work if someone created an EventPosition from code. For now, we will leave it as is.